src/main/java/edu/ucla/library/iiif/fester/verticles/ManifestVerticle.java

```diff
@@ -581,7 +576,8 @@ private void extractWorkMetadata(final String[] aRow, final CsvHeaders aHeaders,
 
         // Create a brief work manifest for inclusion in the collection manifest
         if (id != null && label != null) {
-            final Collection.Manifest manifest = new Collection.Manifest(IDUtils.encode(myHost, id), label);
+            final Collection.Manifest manifest = new Collection.Manifest(IDUtils.getResourceURI(myHost, id).toString(),
```

Comment:
At the point where the line is wrapping because of nested methods, I tend to think it's more readable to break it out. So, for instance, put `IDUtils.getResourceURI` into a `resourceURI` (or something) variable and pass that variable to new `Collection.manifest()`.